.travis.yml

```diff
@@ -9,7 +9,7 @@ git:
 os:
   - linux
```

Comment:
Do we want to test on osx?

Reply:
I don't think it matters in this case, since we're testing a java executable (which is portable as long as the consumer has a valid JDK)